Update restaurant.json - Wrong key value 'bar&grill' (tag cuisine)

mutipg opened the pull request:
Wrong key value 'bar&grill' changed to 'bar_and_grill' (tag cuisine)

Cj-Malone commented:
`bar&grill` has way more uses.

https://taginfo.openstreetmap.org/keys/cuisine#values

mutipg commented:
This is due to a mistake in the wiki, which I have already corrected. Key values ​​should not contain special characters, and 'ampersand' is one of them. I have also changed all occurrences of bar&grill in the osm data to bar_and_grill.

Snowysauce commented:
This is tricky. I agree that the changed value is in line with general OSM wiki guidance, but it's also a change I would have preferred to see happen organically (i.e., the cuisine wiki page gets updated, the change sticks, and then the values gradually change over as editors consult the page for guidance). At the same time, I realize that keeping the previous value in the NSI presets doesn't encourage that at all, and actually encourages editors to switch back to the previous value.

I suppose this can be merged if we get to the time of the next release and the edit to the cuisine wiki page hasn't been reverted,

EvanCarroll commented:
What i'm confused about here is "grill" is explicitly documented as "bar and grill"

> A restaurant serving grilled food. Sometimes called a "Bar & Grill" in the United States.	

Why not just delete both "bar&grill" and "bar_and_grill" and rename to grill with `alcohol=yes` and `drink=beer`.

That they serve alcohol is totally irrelevant to the cuisine.

Snowysauce commented:
The changes to the wiki page and the OSM data are still in place as of this writing, and since the next release is imminent, I'll merge this PR.

> Why not just delete both "bar&grill" and "bar_and_grill" and rename to grill with `alcohol=yes` and `drink=beer`.

This could be a discussion worth having, but since there are almost 700 uses of `cuisine=bar_and_grill` (either by itself or in combination with something else), a better venue for the discussion would probably be something like the OSM discussion forums.
